src/sampleutil.cpp

```diff
@@ -419,3 +419,19 @@ void SampleUtil::copyMultiToStereo(CSAMPLE* _RESTRICT pDest, const CSAMPLE* _RES
         pDest[i * 2 + 1] = pSrc[i * numChannels + 1];
     }
 }
+
+
+// static
+void SampleUtil::reverse(CSAMPLE* pBuffer, int iNumSamples) {
+    CSAMPLE temp1, temp2;
+    for (int j = 0; j < iNumSamples / 4; ++j) {
```

Comment:
I think putting temp1 and temp2 inside the loop will help the compiler optimize this